packages/cli/src/endo.js

```diff
@@ -516,69 +488,79 @@ export const main = async rawArgs => {
     .command('state')
     .description('prints the state directory path')
     .action(async _cmd => {
+      const { statePath } = await import('./locator.js');
```

Comment:
With this change, the CLI no longer depends on anything of substance so it starts very quickly. So, fast commands can be fast.